### PR TITLE
Tackle-9 Get Assessment Multilanguage support

### PR DESCRIPTION
**Pull request by jonathanvila**
Issue : https://github.com/konveyor/tackle-pathfinder/issues/58

Added capability to return a translated Assessment on GetAssessment endpoint,  regarding : 
* category name
* question name and description
* option name

User comments are not translated.

*NOTE*
For convenience, once it is deployed or runned, there's the file `test_translations.sql` that can be executed against the db in order to create 2 translations for every element. These translations will be ES, IT, and the text translated will be the original prefixed with the language code.

### CODE REVIEW

File: src/main/java/io/tackle/pathfinder/controllers/AssessmentsResource.java [hunk 1]
```diff
@@ -53,33 +57,35 @@ public Response createAssessment(@QueryParam("fromAssessmentId") Long fromAssess
...
+    assessmentSvc.deleteAssessment(assessmentId);
     return Response.ok().status(Response.Status.NO_CONTENT).build();
   }
 
   @POST
   @Path("risks")
   @Produces("application/json")
   @Consumes("application/json")
-  public List<RiskLineDto> getIdentifiedRisks(@NotNull @Valid List<ApplicationDto> applicationList) {
+  public List<RiskLineDto> getIdentifiedRisks(@NotNull @Valid List<ApplicationDto> applicationList, @QueryParam("language") String language) {
```
Comment (m-brophy) [inline src/main/java/io/tackle/pathfinder/controllers/AssessmentsResource.java:85]: language parameter isn't used

File: src/test/java/io/tackle/pathfinder/controllers/AssessmentsResourceTest.java [hunk 2]
```diff
@@ -48,7 +48,9 @@
 @QuarkusTestResource(value = KeycloakTestResource.class,
         initArgs = {
                 @ResourceArg(name = KeycloakTestResource.IMPORT_REALM_JSON_PATH, value = "keycloak/quarkus-realm.json"),
-                @ResourceArg(name = KeycloakTestResource.REALM_NAME, value = "quarkus")
+                @ResourceArg(name = KeycloakTestResource.REALM_NAME, value = "quarkus"),
+				@ResourceArg(name = KeycloakTestResource.IMAGE_TAG, value = "latest")
```
Comment (m-brophy) [inline src/test/java/io/tackle/pathfinder/controllers/AssessmentsResourceTest.java:52]: is this image_tag parameter needed?

File: src/test/java/io/tackle/pathfinder/services/TranslatorSvcTest.java [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+            .filter(e -> e.getOrder() == 1)
+            .findFirst()
+            .get().getQuestions().stream()
+            .filter(a -> a.getOrder() == 1)
+            .findFirst().get().getOptions().stream()
+            .filter(b -> b.getOrder() == 1)
+            .findFirst().get();
+        assertThat(assessmentOptionDtoES.getOption()).startsWith("ES:");
+
+        // Checking a language not existing, should give the original texts
```
Comment (m-brophy) [inline src/test/java/io/tackle/pathfinder/services/TranslatorSvcTest.java:84]: comment needs to be updated to match code

**Review by m-brophy — CHANGES_REQUESTED** [review]

File: .github/workflows/test-minikube.yaml [hunk 4]
```diff
@@ -10,15 +10,15 @@ jobs:
...
+        uses: manusa/actions-setup-minikube@v2.4.2
         with:
-          minikube version: 'v1.16.0'
-          kubernetes version: 'v1.19.2'
+          minikube version: v1.21.0
+          kubernetes version: 'v1.21.0'
           github token: ${{ secrets.GITHUB_TOKEN }}
           driver: docker
-          start args: '--addons=registry --addons=ingress --cpus 2 --memory 6000'
+          start args: ' --force --addons=ingress --cpus 2 --memory 6000'
```
[the remark this answers is not in the document]
Comment (jonathanvila) [reply .github/workflows/test-minikube.yaml:21]: :+1 removing it.

File: src/main/java/io/tackle/pathfinder/controllers/AssessmentsResource.java (hunk 1 above)
Comment (jonathanvila) [reply src/main/java/io/tackle/pathfinder/controllers/AssessmentsResource.java:85]: :+1: good catch

File: src/test/java/io/tackle/pathfinder/controllers/AssessmentsResourceTest.java (hunk 2 above)
Comment (jonathanvila) [reply src/test/java/io/tackle/pathfinder/controllers/AssessmentsResourceTest.java:52]: :+1:

**Review by m-brophy — APPROVED** [review]

**Pull request merged**